packages/block-library/src/media-text/index.js

```diff
@@ -156,16 +162,22 @@ export const settings = {
 			mediaUrl,
 			mediaWidth,
 			mediaId,
+			imageFill,
+			focalPoint,
 		} = attributes;
+		const imageFillStyle = imageFillStyles( mediaUrl, focalPoint );
 		const mediaTypeRenders = {
-			image: () => <img src={ mediaUrl } alt={ mediaAlt } className={ ( mediaId && mediaType === 'image' ) ? `wp-image-${ mediaId }` : null } />,
+			image: () => imageFill ?
+				<div style={ imageFillStyle } ></div> :
```

Comment:
Is it necessary to change the markup?
Keeping the image inline and using `object-fit` should accomplish the same thing:
```
.wp-block-media-text__media > img {
    height: 100%;
    object-fit: cover;
    object-position: 50% 50%;
}
```

Reply:
When allowing the image to fill the entire component, it can lead to large parts of the image being cropped. Therefore we need to provide the user with an opportunity to select which part of the image is most important so it stays in view. For this we're making use of the `FocalPointPicker` component here which currently operates by changing the `background-position` inline.

Reply:
Gotcha. background-image works. :smile: 
Object-fit does the same but allows for all the optimizations that can be done with inline images.

It's definitely outside the scope of this PR and this component but maybe in a post-IE future we can look at adding `object-position` or better yet an `--image-position` CSS var to the `FocalPointPicker`.